### ISSUE TITLE
Error when using with three 0.149.0

### ISSUE THREAD

**Issue by sanjaytwisk** [splinetool/r3f-spline#15]
The error seems to be coming from SplineLoader, but since that is not an open source library I decided to raise the issue here.

Error:
```
Uncaught Error: Could not load https://prod.spline.design/IcnOpHOp4rFkJJie/scene.splinecode: Cannot read properties of undefined (reading 'x'))
    at index-7dbf758c.esm.js:1586:36
    at Object.onLoad (SplineLoader.js:1338:75098)
```

Codesandbox:
https://codesandbox.io/p/sandbox/untitled-copy-forked-4qc4wv?selection=%5B%7B%22endColumn%22%3A1%2C%22endLineNumber%22%3A14%2C%22startColumn%22%3A1%2C%22startLineNumber%22%3A14%7D%5D&file=%2Findex.html

**Comment by damnitjoshua** [comment]
I am having this issue too. A temporary fix is to downgrade three to 0.138.3.

**Comment by iambarnaby** [comment]
> I am having this issue too. A temporary fix is to downgrade three to 0.138.3.

same here. This worked with getting it working with chrome and safari but not in firefox. If anyone has a permanent fix that would be appreciated.

**Comment by sanjaytwisk** [comment]
@iambarnaby I'm using three `0.142.0` which works in both chrome and firefox for me

**Comment by marcofugaro** [comment]
Fixed in `@splinetool/loader` version `0.9.226`
